emissary.yaml

```
@@ -0,0 +1,115 @@
+      output: apiext
+
+  - uses: go/build
+    with:
+      packages: ./cmd/busyambassador
+      output: busyambassador
+
+  - uses: go/build
+    with:
+      packages: ./cmd/capabilities_wrapper
+      output: wrapper
+
+  - name: wrapper-setcap
+    # See https://github.com/emissary-ingress/emissary/blob/ac2dc64c6621cd8ec5617f3328544364bdd3fb01/build-aux/Dockerfile#L129-L137
+    runs: |
+      setcap cap_net_bind_service=p ${{targets.contextdir}}/usr/bin/wrapper
+
+  - uses: go/build
+    with:
+      packages: ./cmd/kubestatus
+      output: kubestatus
+
+  - name: Python Build
+    runs: |
+      # Recommendation is to use the host pip instead of using venv.
+      # Pip will complain about being run as root - this disables that warning.
+      export PIP_ROOT_USER_ACTION=ignore
+
+      # Install build time dependencies.
+      pip install -r python/requirements.txt
+      pip install -r python/requirements-dev.txt
+
+      # Install to target dir.
+      # Ignore installed packages, since this looks at the builder host packages, when we're trying to install
+      # the necessary packages into the target directory.
+      pip install --root ${{targets.contextdir}} --ignore-installed -r python/requirements.txt
+
+      # Generate version file - this static file is read by programs to provide version information in logs.
+      # See https://github.com/emissary-ingress/emissary/blob/ac2dc64c6621cd8ec5617f3328544364bdd3fb01/python/ambassador/VERSION.py#L26
+      make python/ambassador.version
+      cp -p python/ambassador.version ${{targets.contextdir}}/usr/lib/python3.12/site-packages
+
+      # Install Emissary Python package
+      cd python
+      python setup.py install --prefix=/usr --root="${{targets.contextdir}}"
+
+      # Install templates - this is not a python package, so it's not installed by the steps above.
+      # This contains a set of HTML templates used by the Python code, but is read relative to the module.
+      # See https://github.com/emissary-ingress/emissary/blob/ac2dc64c6621cd8ec5617f3328544364bdd3fb01/python/ambassador_diag/diagd.py#L603
+      cp -a templates ${{targets.contextdir}}/usr/lib/python3.12/site-packages/templates
+
+      cp -p entrypoint.sh ${{targets.contextdir}}/usr/bin/entrypoint.sh
+      chmod +x ${{targets.contextdir}}/usr/bin/entrypoint.sh
+
+update:
+  enabled: true
+  github:
+    identifier: emissary-ingress/emissary
+    strip-prefix: v
+    use-tag: true
+  ignore-regex-patterns:
+    - -rc.*
+    - -dev.*
+    - ^chart/
+
+test:
+  environment:
+    contents:
+      packages:
+        - curl
+  pipeline:
+    - uses: python/import
+      with:
+        imports: from ambassador import Version
+    - uses: test/kwok/cluster
+    - runs: |
+        /usr/bin/entrypoint.sh &
+
+        # Wait for server to come up
+        sleep 10
+
+        curl -i http://127.0.0.1:8004/ambassador/v0/check_alive
+        curl -i http://127.0.0.1:8004/ambassador/v0/check_ready
```

Let's use `test/daemon-check-output` here:

```suggestion
    - name: "Test entrypoint"
      uses: test/daemon-check-output
      with:
        start: /usr/bin/entrypoint.sh
        timeout: 30
        expected_output: |
          expected_log1
          expected_log2
          expected_log3
        post: |
          curl -i http://127.0.0.1:8004/ambassador/v0/check_alive
        curl -i http://127.0.0.1:8004/ambassador/v0/check_ready
```

---

Done, though had to modify the error strings a bit since there are some "error" logs being output due to the binary not running in a pod.